### PR TITLE
Create Get-TeamTargetingHierarchyStatus.md

### PR DESCRIPTION
**Pull request by ayrtonsoftware**

### CODE REVIEW

File: teams/teams-ps/teams/Get-TeamTargetingHierarchyStatus.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+Get-TeamTargetingHierarchyStatus [-RequestId <String>]
+```
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Set-TeamTargetingHierarchy -FilePath d:\hier.csv
+
+Key       Value
```
Comment (tseward) [inline teams/teams-ps/teams/Get-TeamTargetingHierarchyStatus.md:29]: Output from a cmdlet should be encased in ```output:

\```powershell
\> Cmdlet
\```

\```output
Result
\```

File: teams/teams-ps/teams/Get-TeamTargetingHierarchyStatus.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+schema: 2.0.0
+author: mbergamo
+ms.author: mbergamo
+ms.reviewer:
+---
+# Get-TeamTargetingHierarchyStatus
+
+## SYNOPSIS
+
+Get the status of a hierarchy upload (see [Set-TeamTargetingHierarchy](https://docs.microsoft.com/powershell/module/teams/get-teamtargetinghierarchy))
```
Comment (tseward) [inline teams/teams-ps/teams/Get-TeamTargetingHierarchyStatus.md:14]: Can you move the reference to `Set-TeamTargetingHierarchy` to the Related Links section?

**Review by tseward — CHANGES_REQUESTED** [review]

File: teams/teams-ps/teams/Get-TeamTargetingHierarchyStatus.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+---
+external help file: Microsoft.Open.Teams.CommonLibrary.dll-Help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/powershell/module/teams/connect-microsoftteams
+schema: 2.0.0
+author: mbergamo
```
Comment (get-itips) [inline teams/teams-ps/teams/Get-TeamTargetingHierarchyStatus.md:6]: author metadata should be a github username

**Review by get-itips — CHANGES_REQUESTED** [review]
small suggestions and also see Trevor's

**Comment by dariomws** [comment]
If you have a Microsoft alias, you can link your accounts in GitHub so that a blue "Microsoft submitter" label appears to any PR's or issue you submit. Guidance on it here: https://docs.opensource.microsoft.com/tools/github/index.html

Here is the site you can use to see the mappings between GitHub and Microsoft aliases: https://repos.opensource.microsoft.com/people?q=ayrtonsoftware

**Review by dariomws — APPROVED** [review]
I'm ok after suggestions.

**Comment by yogkumgit** [comment]
@ayrtonsoftware

**Comment by yogkumgit** [comment]
Gentle reminder @ayrtonsoftware Please implement the suggestions. Thanks

**Comment by yogkumgit** [comment]
@Kateyanne Please help.

**Comment by Kateyanne** [comment]
@ayrtonsoftware please review the comments and implement the suggestions. Then we can push this to live.

**Pull request merged**